### DISCUSSION TITLE
Show average difficulty (star rating) on beatmaps

### DISCUSSION THREAD

**Discussion by TheBrunoRM** [ppy/osu#17104] [Ideas]
Similar to BPM, it would be cool to show the average star rating of a beatmap along with its star rating.
Something like this: 8.32* (6.34*)
It could also be displayed with the average first, then the maximum or spike: 6* (8* max)
I was thinking that combo patterns could be used to calculate difficulty, then average them.
There are some 7-8+ star beatmaps that are mostly just 6-star maps with a difficulty spike that makes them 8+ stars. Therefore, I think this could be useful.

**Comment by peppy** [comment]
That's not really how star rating works as a number currently. I think it would require a complete rethinking of how the number works.
